### PR TITLE
Remove references to deprecated rush service.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
@ewiseblatt please review.
@lwander This touches the k8s setup as well.
@tomaslin This touches the docker-compose setup.

File: experimental/packer/configure.sh [hunk 1]
```diff
@@ -130,8 +130,6 @@ gem install bundler
 
 #deb-s3 upload --bucket my-bucket my-deb-package-1.0.0_amd64.deb
 
-docker run -d -p 5000:5000 --name registry registry:2
```
Comment (ewiseblatt) [inline experimental/packer/configure.sh]: Is this change for some other reason? Ok, just curious.

**Comment by ewiseblatt** [comment]
LGTM assuming port 5000 reference was intentional and that was the only occurance.

**Comment by lwander** [comment]
LGTM, thanks for doing the k8s setup as well

File: experimental/packer/configure.sh (hunk 1 above)
Comment (duftler) [inline experimental/packer/configure.sh]: That related to the (deprecated) mechanism for running jobs within rush as docker containers.

**Pull request merged**